docs/FAQ.md

```diff
@@ -355,18 +355,19 @@ for the RAPIDS plugin.
 
 ### Are table layout formats supported?
 
-Yes, there is GPU support for [Delta Lake](./additional-functionality/delta-lake-support.md) and
-[Apache Iceberg](./additional-functionality/iceberg-support.md). See the additional support
-documentation for specifics on the operations supported for these formats.
+Yes, there is GPU support for Delta Lake read 
+and [Apache Iceberg 0.13 read](./additional-functionality/iceberg-support.md).  
+Write operations for Delta Lake is supported, but for [Apache Iceberg 0.13 write](./additional-functionality/iceberg-support.md)
+will fall back to the CPU.
```

Comment:
Please remove this change.  It is clearer as written and will direct users who want more details to the specific documentation.  The change proposed here implies we fully (or even mostly) support Delta Lake writes which is not the case.